recipes/bear/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+MACOSX_DEPLOYMENT_TARGET:  # [osx]
```

Comment:
Is this really necessary, or does `CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"` in the build.sh work?

https://github.com/conda-forge/conda-forge.github.io/pull/1547

Reply:
This is not sufficient: 
 no member named 'clock_gettime' in the global namespace

Reply:
clock_gettime requires SDK 10.12+